Sprint-1/interpret/time-format.js

```diff
@@ -12,13 +12,21 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+// 5
 
 // b) How many function calls are there?
+// 1
 
 // c) Using documentation, explain what the expression movieLength % 60 represents
+// calculate the remainder when movieLength is divided by 60
 
 // d) Interpret line 4, what does the expression assigned to totalMinutes mean?
+// calculate total minutes by subtracting the remaining seconds from the total seconds and dividing it by 60
```

Comment:
Can we calculate `totalMinutes` as `movieLength / 60` (without subtracting `remainingSeconds`?